README.md

```diff
@@ -13,8 +13,19 @@ npm start
 ## Testing
 
 ### Integration tests
+You need to run Conductor locally. The easiest 
+way to get Conductor up and running quickly is using
+docker-compose like this:
 ```
-npm run intergration
```

Comment:
Do not delete `npm run integration`
What you are adding are not integration tests but end to end tests. Can you add a section in readme on how to start them? E.g. `npm run e2e` would be nice.